blueprints/ember-bootstrap/index.js

```diff
@@ -6,60 +6,151 @@ const fs = require('fs');
 const path = require('path');
 const writeFile = rsvp.denodeify(fs.writeFile);
 const chalk = require('chalk');
+const BuildConfigEditor = require('ember-cli-build-config-editor');
+const SilentError = require('silent-error'); // From ember-cli
```

Comment:
It's part of ember-cli, which is why I didn't declare it as a dependency. It displays the error message in red without the stack trace.